scripts/foosbot.coffee

```diff
@@ -694,6 +695,60 @@ returnFromCleanseRespond = (res) ->
     res.reply "Welcome back! Now go kick some ass"
 
 
+getTeamStats = (playerOneName, playerTwoName) ->
+    stats = {
+        "wins": 0,
+        "losses": 0,
+        "goalsFor": 0,
+        "goalsAgainst": 0
+    }
+
+    for finishedGame in finishedGames
+        teamOne = [finishedGame["team1"]["player1"], finishedGame["team1"]["player2"]]
+        teamTwo = [finishedGame["team2"]["player1"], finishedGame["team2"]["player2"]]
+
+        if playerOneName in teamOne && playerTwoName in teamOne
+            myTeam = finishedGame["team1"]
+            otherTeam = finishedGame["team2"]
+        else if playerOneName in teamTwo && playerTwoName in teamTwo
+            myTeam = finishedGame["team2"]
+            otherTeam = finishedGame["team1"]
+        else
+            continue
+
+        if myTeam["score"] > otherTeam["score"]
+            stats["wins"] += 1
+        else
+            stats["losses"] += 1
+
+        stats["goalsFor"] += myTeam["score"]
+        stats["goalsAgainst"] += otherTeam["score"]
+
+    return stats
+
+
+teamStatsRespond = (res) ->
+    playerOneName = if res.match[1] == 'me' then res.message.user.name else res.match[1].trim().lower()
+    playerTwoName = if res.match[2] == 'me' then res.message.user.name else res.match[2].trim().lower()
+
+    if playerOneName == playerTwoName
+        res.send "Player one and player two cannot be the same."
+        return
+
+    stats = getTeamStats(playerOneName, playerTwoName)
```

Comment:
Maybe call the variable `teamStats`, since other places in the code create a `stats` variable but it refers to a single player's stats.